pykechain/models/widgets/widgets_manager.py

```diff
@@ -540,10 +563,12 @@ def add_filteredgrid_widget(
             }
         )
 
+        meta, title = _set_title(meta, title=title, **kwargs)
+
         if parent_instance_id:
             meta[AssociatedObjectId.PARENT_INSTANCE_ID] = parent_instance_id
-
-        meta, title = _set_title(meta, title=title, **kwargs)
+        if parent_model_id:
+            meta[AssociatedObjectId.PARENT_MODEL_ID] = parent_model_id
```

Comment:
parent_model_id should always be provided, hence the if statement should be removed.

Reply:
Agree